build/ci/phase-template.yml

```diff
@@ -34,9 +34,18 @@ phases:
       displayName: Download Benchmark Data
     - ${{ if eq(parameters.name, 'MacOS') }}:
       - script: brew update && brew install libomp mono-libgdiplus gettext && brew link gettext --force
-        displayName: Install runtime dependencies
+        displayName: Install runtime dependencies     
+    - script: $(_buildScript) -$(_configuration) -runtests -coverage=true
+      displayName: Run Tests with code coverage.
+      condition: and(eq(variables._phaseName, 'Windows_x64'), eq(variables._configuration, 'Debug'))
```

Comment:
I don't think this should be the condition. Instead, we should have a parameter specifically for "analyze code coverage". and then this is based on that parameter. That way we can set the parameter on any leg(s) we'd like.